service/src/main/java/de/starwit/service/impl/DecisionService.java

```diff
@@ -92,6 +93,10 @@ public DecisionEntity createNewDecisionBasedOnIncidentMessage(IncidentMessage de
         ZonedDateTime dateTime = Instant.ofEpochMilli(decisionMessage.getTimestampUtcMs())
                 .atZone(ZoneId.systemDefault());
         entity.setAcquisitionTime(dateTime);
+        if (decisionMessage.getCameraLocation() != null) {
```

Comment:
This must probably (according to doc, haven't tested it) be `decisionMessage.hasCameraLocation()`, because empty fields are always filled with defaults as a protobuf convention. See documentation: https://protobuf.dev/reference/java/java-generated/#singular-proto3